package.json

```diff
@@ -1,9 +1,11 @@
 {
   "name": "contexture-elasticsearch",
-  "version": "0.1.2",
+  "version": "0.1.3",
   "description": "ElasticSearch Provider for Contexture",
   "main": "src/index.js",
   "scripts": {
+    "prepublish": "./node_modules/.bin/metagen src/example-types commonJS ./___all.js && cat src/example-types/___all.js | sed '1d;$d' | sed 's/return/module.exports = /' > src/example-types/__all.js && rm src/example-types/___all.js",
```

Comment:
let;s not use a sed script, let's add a new format to directory-metagen (and rename the current commonJS to amd-commonJS